content/en/tracing/profiler/profiler_troubleshooting.md

```diff
@@ -121,5 +128,77 @@ Override templates let you specify profiling properties to override. However, th
 
 {{< partial name="whats-next/whats-next.html" >}}
 
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+
+## Missing profiles in the profile search page
+
+If you've configured the profiler and don't see profiles in the profile search page, turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:
+
+- Operating system type and version (for example, Linux Ubuntu 20.04)
+- Runtime type, version, and vendor (for example, Python 3.9.5)
+
+[1]: /tracing/troubleshooting/#tracer-debug-logs
+[2]: /help/
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="go" >}}
+
+## Missing profiles in the profile search page
+
+If you've configured the profiler and don't see profiles in the profile search page, turn on [debug mode][1] and [open a support ticket][2] with debug files and the following information:
+
+- Operating system type and version (for example, Linux Ubuntu 20.04)
+- Runtime type, version, and vendor (for example, Go 1.16.5)
+
+[1]: /tracing/troubleshooting/#tracer-debug-logs
+[2]: /help/
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+## Missing profiles in the profile search page
```

Comment:
Remove this section, duplicated with the above Missing profiles section

Reply:
This is only semi-duplicated: I took the chance to change the example to match every language:

> * Runtime type, version, and vendor (for example, Java OpenJDK 11 AdoptOpenJDK)
> * Runtime type, version, and vendor (for example, Python 3.9.5)
> * Runtime type, version, and vendor (for example, Go 1.16.5)
> * Runtime type, version, and vendor (for example, Ruby 2.7.3)

...although I wonder if I should also remove the mention to "vendor" as that's usually not relevant outside of Java.